src/widgets.zsh

```diff
@@ -47,6 +47,11 @@ _zsh_autosuggest_accept() {
 	_zsh_autosuggest_invoke_original_widget $@
 }
 
+# Accept the entire suggestion and execute it
+_zsh_autosuggest_execute() {
```

Comment:
I believe we still need to [set the buffer](https://github.com/tarruda/zsh-autosuggestions/pull/122#discussion-diff-53573072R55) and [clear the postdisplay](https://github.com/tarruda/zsh-autosuggestions/pull/122#discussion-diff-53573072R58) here before accepting the line.